sktime/transformations/series/detrend/_deseasonalize.py

```diff
@@ -25,13 +23,39 @@
 class Deseasonalizer(_SeriesToSeriesTransformer):
     """A transformer that removes seasonal components from time series.
 
+    Fit computes seasonal components and stores them in `_seasonal`.
```

Comment:
does _seasonal refer to the attribute seasonal_?

Reply:
True, good catch. Thanks @RHKuhns